src/FakeItEasy/StrictFakeOptions.cs

```diff
@@ -0,0 +1,37 @@
+namespace FakeItEasy
+{
+    using System;
+
+    /// <summary>
+    /// Options for strict fakes.
+    /// </summary>
+    [Flags]
+    public enum StrictFakeOptions

```

Comment:
Maybe we could find a better name for this. It's unlikely that we'll ever add new options for strict fakes, and if we do, an enum might not be enough. We *could* rename it to `AllowedMembers` or `StrictFakeAllowedMembers`...
I initially named it `StrictFakeBehavior`, but FxCop reminded me that a `[Flags]` enum should have a plural name, and `StrictFakeBehaviors` didn't sound right.

Reply:
…Behaviors certainly isn't common usage, but I don't think it's the worst thing in the world.
Truth be told, I don't really understand your concern here. You seem to be worried both about the name and the inflexibility of the enum. For the latter, we could always implement an options-building scheme like the "outer" options, but it seems heavy.

I'm not particularly worried about what you have done, but am prepared to be convinced to worry.

Reply:
I'm not really worried about the inflexibility of the enum, it's just that if we ever need to add more options for which an enum isn't a good fit, the name will already be taken.

> For the latter, we could always implement an options-building scheme like the "outer" options, but it seems heavy.

Yeah... probably overkill, especially for something we don't expect to ever need

If you're not worried, I'm not worried!